### PR TITLE
Mask counters for production

### PR DESCRIPTION
**Pull request by YannickSoehngen**
Added missing set method in MatchMaker and fixed HitMaker for masking counters

### CODE REVIEW

File: StRoot/StETofHitMaker/StETofHitMaker.cxx [hunk 1]
```diff
@@ -1252,14 +1252,16 @@ StETofHitMaker::matchSides()
...
 
 	    //Modify individual counters if needed (e.g. flip in local y due to switched cables)
 	    if(mModMatrix.at(detIndex) > 0){
 	      int mode = mModMatrix.at(detIndex);
 	      modifyHit(mode, posX , posY , time);
 	    }
 	    
+	    
+            StETofHit* constructedHit = new StETofHit( sector, plane, counter, time, totSum, clusterSize, posX, posY );
+
```
Comment (plexoos) [inline StRoot/StETofHitMaker/StETofHitMaker.cxx]:
```suggestion
            StETofHit* constructedHit = new StETofHit( sector, plane, counter, time, totSum, clusterSize, posX, posY );
```

**Review by plexoos — COMMENTED** [review]
Am I correct in understanding that this resolves a bug that occurred when the `modifyHit(mode, posX, posY, time)` function was called after a `StETofHit` was created?

File: StRoot/StETofHitMaker/StETofHitMaker.cxx (hunk 1 above)
Comment (YannickSoehngen) [reply StRoot/StETofHitMaker/StETofHitMaker.cxx]: Yes, thats correct

**Review by plexoos — APPROVED** [review]

**Comment by plexoos** [comment]
@PhilippWeidenkaff can you review the change please?

**Comment by plexoos** [comment]
shall we wait or shall we merge @fgeurts ?

**Comment by fgeurts** [comment]
Let's merge. Thanks for following up, @plexoos

**Review by fgeurts — APPROVED** [review]

**Pull request merged**